html/base/inputs/HugeRadio.stories.js

```diff
@@ -0,0 +1,917 @@
+import { markdownDocumentationLinkBuilder } from '../../../storybook-utilities/markdownDocumentationLinkBuilder';
+import { useEffect } from '@storybook/client-api';
+import { radioInput } from './radioInput';
+
+export default {
+  title: 'Components/Input/Huge Radio',
+  decorators: [
+    story => `<div class="sprk-o-Box">${story()}</div>`,
+  ],
+  parameters: {
+    info: `
+${markdownDocumentationLinkBuilder('input')}
+    `,
+    docs: { iframeHeight: 140 },
+  },
+};
+
+export const layoutTwo = () => {
+  useEffect(() => {
+    radioInput();
+  }, []);
+
+  return `
+    <div class="sprk-b-InputContainer sprk-u-MaxWidth--100">
+      <fieldset class="sprk-b-Fieldset">
+        <legend class="sprk-b-Legend">
+          <label class="sprk-b-Label">
+            Radio Group Label
+          </label>
+        </legend>
+
+        <div class="sprk-o-Stack sprk-o-Stack--split@s">
+          <div class="sprk-o-Stack__item sprk-o-Stack__item--flex@s">        
+            <div
+              class="
+                sprk-b-SelectionContainer
+                sprk-b-Radio
+                sprk-b-Radio--huge
+                sprk-u-phs
+              "
+              data-sprk-input="radio"
+            >
+              <input
+                class="sprk-b-Radio__input"
+                id="radio-1"
+                data-id="radio-1"
+                type="radio"
+                name="radio"
+                aria-describedby="radio--error-container"
+              >
+
+              <label
+                for="radio-1"
+                class="sprk-b-Label sprk-b-Label--inline sprk-b-Radio__label"
+              >
+                Radio Item 1
+              </label>
+            </div>
+          </div>
+
+          <div class="sprk-o-Stack__item sprk-o-Stack__item--flex@s">
+            <div
+              class="
+                sprk-b-SelectionContainer
+                sprk-b-Radio
+                sprk-b-Radio--huge
+                sprk-u-phs
+              "
+              data-sprk-input="radio"
+            >
+              <input
+                class="sprk-b-Radio__input"
+                id="radio-2"
+                data-id="radio-2"
+                type="radio"
+                name="radio"
+                aria-describedby="radio--error-container"
+              >
+
+              <label
+                for="radio-2"
+                class="sprk-b-Label sprk-b-Label--inline sprk-b-Radio__label"
+              >
+                Radio Item 2
+              </label>
+            </div>
+          </div>
+        </div>
+      </fieldset>
+
```

Comment:
remove extra line

Reply:
We have that on all the stories to separate the fieldset from the error container.